### PR TITLE
TELCODOCS-1101: Update must-gather image versions

### PR DESCRIPTION
**Pull request by ogradyp**
https://issues.redhat.com/browse/TELCODOCS-1101 Update must-gather image versions

Applies to OpenShift version : 4.12+

Preview: [Gathering data about specific features](https://54544--docspreview.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data)

Dev review complete/approved by @slintes 
QE review complete/approved by @anna-savina 
Peer review complete/approved @Amrita42 

Thank you.

### CODE REVIEW

File: modules/gathering-data-specific-features.adoc [hunk 1]
```diff
@@ -60,11 +60,11 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:{sandboxed-containers-version}.0`
 |Data collection for {sandboxed-containers-first}.
 
-|`registry.redhat.io/workload-availability/self-node-remediation-must-gather-rhel8:v0.4.0`
-|Data collection for the Self Node Remediation Operator and the Node Health Check Operator.
+|`registry.redhat.io/workload-availability/self-node-remediation-must-gather-rhel8:<installed-version-SNR>`
```
Comment (anna-savina) [inline modules/gathering-data-specific-features.adoc:63]:
```suggestion
|`registry.redhat.io/workload-availability/self-node-remediation-must-gather-rhel8:v<installed-version-SNR>`
```

File: modules/gathering-data-specific-features.adoc [hunk 2]
```diff
@@ -60,11 +60,11 @@ endif::openshift-dedicated[]
...
+|Data collection for the Self Node Remediation (SNR) Operator and the Node Health Check (NHC) Operator.
 
-|`registry.redhat.io/workload-availability/node-maintenance-must-gather-rhel8:v{product-version}.0`
-|Data collection for the Node Maintenance Operator.
+|`registry.redhat.io/workload-availability/node-maintenance-must-gather-rhel8:<installed-version-NMO>`
```
Comment (anna-savina) [inline modules/gathering-data-specific-features.adoc:66]:
```suggestion
|`registry.redhat.io/workload-availability/node-maintenance-must-gather-rhel8:v<installed-version-NMO>`
```

**Review by slintes — COMMENTED** [review]
lgtm when Anna's suggestions are applied, for ensuring users use the `v` prefix in the version

File: modules/gathering-data-specific-features.adoc (hunk 1 above)
Comment (ogradyp) [reply modules/gathering-data-specific-features.adoc]: Thanks @anna-savina I have applied your suggestion.

File: modules/gathering-data-specific-features.adoc (hunk 2 above)
Comment (ogradyp) [reply modules/gathering-data-specific-features.adoc]: Thanks @anna-savina I have applied your suggestion.

**Comment by ogradyp** [comment]
> lgtm when Anna's suggestions are applied, for ensuring users use the `v` prefix in the version

Thanks @slintes I have applied Anna's suggestion.

**Comment by ogradyp** [comment]
> /lgtm

Thanks @anna-savina

**Comment by Amrita42** [comment]
lgtm

**Review by jldohmann — APPROVED** [review]

**Comment by ogradyp** [comment]
Thank you Jesse. Much appreciated.

**Pull request merged**